.travis.yml

```diff
@@ -17,6 +17,7 @@ before_install:
   - mongocryptd/bin/mongocryptd&
   - wget https://s3.amazonaws.com/mciuploads/libmongocrypt/ubuntu1604/master/1c6c72377582ebcd8402f6f15fff5c3143b46741/libmongocrypt.tar.gz
   - sudo tar -xzf libmongocrypt.tar.gz -C /usr/local
+  - sudo pip install mongo-orchestration && mongo-orchestration start
```

Comment:
I do git in `check_mongo_orchestration.js` because that allows me to use the `mo` script. Here I did `pip` because it's  sufficient to run `mongo-orchestration start`. For more detail see documentation: https://github.com/10gen/mongo-orchestration